tests/ConnectionTest.php

```diff
@@ -61,6 +63,22 @@ abstract class ConnectionTest extends TestCase
      */
     abstract protected function dropInfrastructure(Connection $connection): PromiseInterface;
 
+    /**
+     * Create database and table.
+     *
+     * @param Connection $connection
+     *
+     * @return PromiseInterface
```

Comment:
`@param` is redundant since there is typehint, `@return` should be made non-redundant by adding the type of the resolved value as `<...>`.